### PR TITLE
Additional info

### PR DESCRIPTION
**Pull request by ghost**
reenable a special field to denote additional information about an engel, ie. languages for translators or professions for doctors.
while we're at it delete hometown and kill some of the tshirts references.

(it's not the cleanest approach, but i was in the files anyways)

### CODE REVIEW

File: includes/view/User_view.php [hunk 1]
```diff
@@ -72,18 +68,15 @@ function Users_view($users, $order_by, $arrived_count, $active_count, $force_act
       )),
       table(array(
           'Nick' => Users_table_header_link('Nick', _('Nick'), $order_by),
-          'Vorname' => Users_table_header_link('Vorname', _('Prename'), $order_by),
+          'Vorname' => Users_table_header_link('Vorname', _('Firstname'), $order_by),
```
Comment (globin) [inline includes/view/User_view.php]: First Name

File: includes/pages/user_settings.php [hunk 2]
```diff
@@ -184,7 +184,7 @@ function user_settings() {
                   form_text('mail', _("E-Mail") . ' ' . entry_required(), $mail),
                   form_checkbox('email_shiftinfo', _("Please send me an email if my shifts change"), $email_shiftinfo),
                   form_text('jabber', _("Jabber"), $jabber),
-                  form_text('hometown', _("Hometown"), $hometown),
+                  form_text('kommentar', _("additional Info(Language / Profession)"), $kommentar),
```
Comment (globin) [inline includes/pages/user_settings.php]: at least (mind the space): **A**dditional Info (Language / Profession)
Although I'd prefer to spell out Information

File: includes/pages/guest_login.php [hunk 3]
```diff
@@ -239,8 +241,11 @@ function guest_register() {
                       )),
                       div('col-sm-6', array(
                           form_text('hometown', _("Hometown"), $hometown) 
-                      )) 
-                  )),
+                      )),
+                      div('col-sm-6', array(
+                              form_text('comment', _("additional Info(Language / Profession)"), $comment)
```
Comment (globin) [inline includes/pages/guest_login.php]: see below

File: includes/pages/user_settings.php [hunk 4]
```diff
@@ -19,7 +19,7 @@ function user_settings() {
   $mail = $user['email'];
   $email_shiftinfo = $user['email_shiftinfo'];
   $jabber = $user['jabber'];
-  $hometown = $user['Hometown'];
+  $kommentar = $user['kommentar'];
```
Comment (globin) [inline includes/pages/user_settings.php]: In english please

File: includes/pages/admin_active.php [hunk 5]
```diff
@@ -9,6 +9,7 @@ function admin_active() {
   
   $msg = "";
   $search = "";
+   //TODO WTF?! do you even count(*) bro?
```
Comment (globin) [inline includes/pages/admin_active.php]: Fix it right away otherwise no one will ever :wink:

Comment (globin) [inline includes/pages/admin_active.php]: Fix it right away otherwise no one will ever :wink:

**Comment by globin** [comment]
Apart from my nitpicking after fixing those LGTM

File: includes/pages/user_settings.php (hunk 4 above)
Comment (ghost) [inline includes/pages/user_settings.php]: the field is actually called like that in the table.

Comment (globin) [inline includes/pages/user_settings.php]: I don't think we should change it there then but nonetheless keep it to English as far as possible. The variable doesn't need to be called `kommentar`, also the other occurrences not directly referencing the database.

**Comment by ghost** [comment]
thanks for the nice suggestions, i hope i catched everything.
maybe it's more like "leave less broken windows" in this codebase ;)

**Pull request merged**